### DISCUSSION TITLE
Make RDKit install visible to pip

### DISCUSSION THREAD

**Discussion by kienerj** [rdkit/rdkit#3996] [Ideas]
A problem I have recently noticed is that when installing a package via pip and setuptools and setup.py (or setup.cfg) has the `install_requires` option set including RDKit, `pip` always fails to detect rdkit even if it is installed and hence aborts installation. Which in the end means rdkit can never be put into the `install_requires` option and hence rdkit dependency must be mentioned somewhere like in the README.

I'm aware `pip` can not install rdkit (or not yet and certainly not on windows) but if it at least could detect a matching version is installed (or not) the user could be correctly warned before installing without rdkit.

Example:

    install_requires =    
        rdkit >=2020.09.1

    ERROR: Could not find a version that satisfies the requirement rdkit>=2020.09.1

While `conda list` contains rdkit in a matching version:

    rdkit                     2021.03.1        py39hfadf033_0

Strangley enough `pip` can easily detect for example `numpy` that was installed via `conda`.

Looking at the `site-packages` folder of a conda environment I immediately noticed a possible cause which I was able to confirm. RDKit upon installation does not create an "installer directory" like `egg-info` or `dist-info`. So this problem can be solved by adding a  `dist-info` directory into `site-packages`. Example: `rdkit-2021.03.1.dist-info`. Turns out that is not enough, The directory need to contain at minimum an empty `METADATA` file and then it works.

So my idea is to add a proper `dist-info` directory upon rdkit installation from conda. The requirements for that can be found [here](https://www.python.org/dev/peps/pep-0427/#the-dist-info-directory).  That way rdkit can be properly used as a requirement in setuptools (pypi) packages.

**Comment by greglandrum** [comment]
Hi @kienerj ,
Thanks for the suggestion. I will take a look!
